assets/css/stripeStyle.css

```diff
@@ -266,6 +260,192 @@ table tr {
 	color: #304ffe;
 }
 
+.toggle {
+	display: none;
+}
+
+.mobile-price-view {
+	display: none;
+}
+
+.mobile-price-view.opened {
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	position: fixed;
+	top: 0;
+	width: 100%;
+	height: 100vh;
+	background: transparent linear-gradient(233deg, #d500f9 0%, #2962ff 99%) 0% 0% no-repeat padding-box;
+	z-index: 100;
+	margin-left: -1rem;
+	padding: 1rem;
+}
+
+.mobile-price-view .nav-back {
+	display: flex;
+    align-items: center;
```

Comment:
Codacy found an issue: [Expected indentation of 1 tab (indentation)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5901386)